lib/sles4sap.pm

```diff
@@ -372,7 +372,8 @@ sub prepare_profile {
 
     if ($has_saptune) {
         assert_script_run 'saptune service takeover';
-        my $ret = script_run("saptune solution verify $profile", die_on_timeout => 0);
+        enter_cmd "saptune solution verify $profile; echo DONE-$\? > /dev/$serialdev";
+        my $ret = wait_serial qr/DONE-\d-/, timeout => 30;
```

Comment:
how `30` has been selected as timeout value?

Reply:
That's the default of `script_run` as used in before.